bioio_imageio/reader_metadata.py

```diff
@@ -21,8 +21,7 @@ def get_supported_extensions() -> List[str]:
         """
         Return a list of file extensions this plugin supports reading.
         """
-        raise NotImplementedError()
-        # return ["ext", "extn"]
+        return ["bmp", "gif", "jpg", "mp4", "png"]
```

Comment:
not sure if we want to include more but here is the full list: https://imageio.readthedocs.io/en/stable/formats/index.html